### ISSUE TITLE
VS Code menu contribution to "view/title" doesn't appear anywhere

### ISSUE THREAD

**Issue by jcortell68** [eclipse-theia/theia#12705]
### Bug Description:
According to the Theia codebase, I should be able to contribute a menu (via a VS Code extension) to the location "view/title". But the menu doesn't appear where I expect it to, or anywhere for that matter.

In VS Code, the menu item appears to the right of the view title, as seen here:

![image](https://github.com/eclipse-theia/theia/assets/109228283/cc9054d1-e9d7-4b97-a37b-d898f49215b0)

Here is Theia code that _seems_ to want to support the menu insertion point:
https://github.com/eclipse-theia/theia/blob/master/packages/plugin-ext/src/main/browser/menus/vscode-theia-menu-mappings.ts#L84

However, looking deeper in the Theia code, there doesn't appear to be any meat behind that declaration.

### Steps to Reproduce:
git clone -b menu_bug_4 https://github.com/jcortell68/theiasandbox
cd theiasandbox
yarn
yarn start:electron

### Additional Information
* Operating System: Windows
* Theia Version: 1.39

**Comment by cdamus** [comment]
I'll be happy to work on this.

**Comment by jcortell68** [comment]
@cdamus I have a question for you as the "side panel" term is a little ambiguous to me. Using the terminology Microsoft came up with for VS Code, a view can appear in one of three areas:

- Primary Sidebar (left of the editor)
- Secondary SIdebar (right of the editor)
- Panel (below the editor)

See https://code.visualstudio.com/api/ux-guidelines/overview

I just want to point out that in VS Code, the `view/title` menu contribution shows in the built-in views regardless of which of these areas the view is located in. Does your Theia fix provide the same behavior?

**Comment by cdamus** [comment]
> I just want to point out that in VS Code, the `view/title` menu contribution shows in the built-in views regardless of which of these areas the view is located in. Does your Theia fix provide the same behavior?

Thanks, @jcortell68 . I am fairly certain that views in the "Panel" (where problems and terminals are) would not get these menu contributions. Would have to check on the "Secondary Sidebar" (which I never use). In any case, I'll have some revision to make in this PR.

**Comment by jcortell68** [comment]
@cdamus That the contribution won't appear in views in the Panel area...is that because of a limitation in Theia, rather than a limitation in your fix?

**Comment by jcortell68** [comment]
Yeah. I'm with you on the "added" contribution in Theia vs VS Code. The example contribution wants it in all views (no `when` specification), so if Theia has some additional view elements that aren't in VS Code...then it should appear in those, too :-)

Thanks for the adjustment!
